config/initializers/azure.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.config.to_prepare do
+  Azure.configure do |config|
+    begin
+      google_cloud_credentials = JSON.parse(ENV.fetch('GOOGLE_CLOUD_CREDENTIALS_STATS', '{}')).deep_symbolize_keys
+    rescue TypeError
+      raise Azure::GoogleCloudCredentialsError
```

Comment:
Can we add a message here for any future us? "No ENV['GOOGLE_CLOUD_CREDENTIALS_STATS'] found."

Reply:
https://github.com/DFE-Digital/apply-for-teacher-training/pull/9689/files#diff-a8f8614a7d8204d996de735cb5a9cd44b12f9477f8df6e3411b16df187a6e450R17-R21